locale/fr/get-involved/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Comment s'impliquer ?
+layout: contribute.hbs
+---
+
+# Nous encourageons toutes les formes de contributions de la part de la communauté 
+
+La communauté Node.js est vaste, inclusive et excitée de permettre à autant 
+d'utilisateurs de contribuer de toutes les manières possibles. Si vous souhaitez
+[signaler un problème](https://github.com/nodejs/node/issues), aider à la documentation
```

Comment:
'aider a ecrire/maintenir la documentation' ?

Reply:
aider à documenter ?

Reply:
Maybe, the way to express needs to be balanced in the whole paragraph.

Reply:
I don't understand your latest comment

Reply:
At the end, reading this paragraph should feel like it was originally written in French.